### PR TITLE
Angular edit story

### PR DESCRIPTION
**Pull request by colinyoung87**

### CODE REVIEW

File: app/assets/javascripts/templates/stories/edit.jst.ejs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <div class="pure-g">
+    <div class="pure-u-1 pure-u-md-17-24 ui-story-container">
+      <div class="ui-story">
+        <article>
+          <header>
+            <div class="actions">
+              <a href="/stories/{{ story.slug }}">Cancel</a>
+            </div>
+            <user-avatar user="story.owner"></user-avatar>
+            {{ story.owner.display_name }} added <human-time datetime="story.created_at"></human-time>
```
Comment (phawk) [inline app/assets/javascripts/templates/stories/edit.jst.ejs]: I think you should be able to edit the story name

**Comment by phawk** [comment]
:+1:

File: app/assets/javascripts/templates/stories/edit.jst.ejs (hunk 1 above)
Comment (colinyoung87) [inline app/assets/javascripts/templates/stories/edit.jst.ejs]: ^^ line 2 :smiley:

Comment (phawk) [inline app/assets/javascripts/templates/stories/edit.jst.ejs]: doh!

**Pull request merged**